packages/push-cordova/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@aerogear/cordova-plugin-aerogear-push",
+  "version": "0.3.0",
```

Comment:
0.3.0?

Reply:
We use single version per entire SDK. New packages will get the version that currently exist.

Reply:
It means all SDK should have the same version?

Reply:
We do not care as this version is being changed by our automation scripts on release. Then we decide what versions all packages should have.